bindata/assets/config/defaultconfig-postbootstrap-highnodeutilization.yaml

```diff
@@ -2,11 +2,16 @@ apiVersion: kubescheduler.config.k8s.io/v1beta2
 kind: KubeSchedulerConfiguration
 profiles:
   - schedulerName: default-scheduler
+    pluginConfig:
+      - args:
+          scoringStrategy:
+            type: MostAllocated
+        name: NodeResourcesFit
     plugins:
       score:
         disabled:
-        - name: "NodeResourcesLeastAllocated"
+        - name: "NodeResourcesFit"
```

Comment:
does this need to be disabled? I think there were changes made so that you don't need to explicitly disable a default plugin to override it (eg, change weight) anymore

Reply:
I am not sure. I was trying to figure that out. The `NodeResourcesFit` plugin can have only a single `pluginConfig` configuration. So what you are saying makes sense.

Reply:
Removed.